.babelrc

```diff
@@ -4,9 +4,10 @@
   "comments": false,
   "presets": [
     "react-native",
+    "react-native-stage-0/decorator-support"
   ],
   "plugins": [
-    "./plugins/babelRelayPlugin"
+    ["relay", {"compat": true, "schema": "data/schema.json"}],
```

Comment:
Can just be ["relay"] unless compatibility mode is needed
https://facebook.github.io/relay/docs/babel-plugin-relay.html